### DISCUSSION TITLE
media and weather

### DISCUSSION THREAD

**Discussion by skafew** [end-4/dots-hyprland#482] [General]
how do i remove this : 
![image](https://github.com/end-4/dots-hyprland/assets/60267797/7ec6d898-32d7-40a0-b4fd-d4649bafee78)
and weather status is not showing for some reason
![image](https://github.com/end-4/dots-hyprland/assets/60267797/56cb8166-4d4c-4513-a5d8-f0da30b00431)

**Comment by end-4** [comment]
> how do i remove this :

mess with `~/.config/ags/modules/bar/normal/music.js`

> weather status isn't showing

it won't show if you're on a device with battery. on a desktop it would show in place of the battery indicator (and those quick action buttons that nobody uses)

**Reply by skafew** [reply]
thank you

**Reply by skafew** [reply]
btw how can i change booru source to danbooru thx

**Reply by end-4** [reply]
the 2 apis available atm, yande.re and konachan, have the same response structure in their apis so it's simple as changing the endpoint. this isn't the case for danbooru
of course i can make a converter but i'm lazy because yande.re just works

**Reply by Fousssx** [reply]
Where would i change the endpoint pls ?
